includes/User.php

```diff
@@ -3026,6 +3026,13 @@ public function doLogout() {
 		$this->clearCookie( 'UserID' );
 		$this->clearCookie( 'Token' );
 
+		// Wikia change - begin (@see PLATFORM-1028)
+		// @author macbre
+		// There's no need to keep the user name (in both session and cookie) when you log out
+		$this->getRequest()->setSessionData( 'wsUserName', null );
+		$this->clearCookie( 'UserName' );
```

Comment:
The cookie name is `wikicitiesUserName` and the prefix seems not be added automatically by `setcookie` methods called later.